package.json

```diff
@@ -112,6 +112,7 @@
     "@elastic/node-crypto": "1.2.1",
     "@elastic/numeral": "^2.5.1",
     "@elastic/react-search-ui": "^1.19.0",
```

Comment:
could you also update all the search-ui related dependencies to 1.19.1 please?

Reply:
yep! looks like they are resolved to `1.19.1` in the `yarn.lock` but i'll make the change here also so that's clear